src/seedu/addressbook/ui/TextUi.java

```diff
@@ -19,12 +19,12 @@
 public class TextUi {
 
     /** A decorative prefix added to the beginning of lines printed by AddressBook */
-    private static final String LINE_PREFIX = "|| ";
+    //private static final String LINE_PREFIX = "|| ";
```

Comment:
remove unnecessary code fully. no need to retain them as comments. 
if you need any part of the code later, you can retrieve it from previous commits :-)